envConfig/stage.env

```diff
@@ -0,0 +1,10 @@
+SIMORGH_BASE_URL=https://www.stage.bbc.com
+SIMORGH_PUBLIC_STATIC_ASSETS_ORIGIN=https://news.test.files.bbci.co.uk
+SIMORGH_PUBLIC_STATIC_ASSETS_PATH=/include/articles/public/
+SIMORGH_ATI_BASE_URL=https://logws1363.ati-host.net?
+SIMORGH_EMBEDS_BASE_URL_TEST=https://polling.test.bbc.co.uk
+SIMORGH_EMBEDS_BASE_URL_LIVE=https://polling.bbc.co.uk
+SIMORGH_MPULSE_API_KEY=JLSEX-R76DK-JXL3A-UB7YH-NPSEW
+LOG_DIR='/var/log/simorgh'
+LOG_LEVEL='info'
+APP_ENV=stage
```

Comment:
Is this required?

Reply:
I would not expect this to ever be used as Simorgh itself has no notion of a stage environment, @hindsc52 that's so right?